account_tax_one_vat/models/account_move_line.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, models
+
+
+class AccountMoveLine(models.Model):
+    _name = "account.move.line"
+    _inherit = ["account.move.line", "one.vat.mixin"]
+
+    @api.onchange("tax_ids")
+    def _onchange_tax_ids(self):
```

Comment:
I don't understand how you handle Intra community vat. When you are in a intra com fiscal position, there are 2 vat.

Reply:
In BE, the fiscal position map a national tax to an intracom tax. It is always a 1-1 mapping

Reply:
Well, not was always the case. 

In fact : 
- in Odoo V12, the mapping is 1 -> 2. [see](https://github.com/odoo/odoo/blob/12.0/addons/l10n_fr/data/account_fiscal_position_template_data.xml#L133) tva_acq_normale -> tva_intra_normale + tva_acq_intra_normale.
- In Odoo V16 , the mapping is 1 -> 1. [See](https://github.com/odoo/odoo/blob/16.0/addons/l10n_fr/data/account_fiscal_position_template_data.xml#L151) tva_acq_normale -> tva_intra_normale_biens.